nemo/collections/asr/models/asrconvctcmodel.py

```diff
@@ -114,7 +119,7 @@ def __instantiate_modules(
 
     @property
     def train_graph(self) -> NeuralGraph:
-        return self.__training_neural_graph
+        return None
```

Comment:
The fact that this is now defaulting to return None shows that we should have thought more about NeMoModels in general.

Reply:
well, the reason I am not doing NeuralGraph in train is because in this particular case, I need to be able to swap decoder if the user choses to fine-tune the model on a different vocabulary.